app/helpers/application_helper.rb

```diff
@@ -41,7 +41,11 @@ def fingerprinted(path, filename, file_prefix = nil)
   def css_fingerprinted(filename)
     manifest_path = "#{Rails.root}/public/assets/javascripts/manifest.json"
     manifest = Oj.load(File.read(File.expand_path(manifest_path, __FILE__)))
-    manifest[filename].split('/').last
+    if manifest[filename]
```

Comment:
This seems unrelated. What is this change for?

Reply:
I'll stash that change, it was for one of the test failing locally for me but i guess it’s unnecessary and was probably a local issue